static/app/components/superuserStaffAccessForm.tsx

```diff
@@ -60,16 +59,17 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
     }
 
     const authenticators = await this.getAuthenticators();
-    this.setState({authenticators: authenticators});
+    this.authenticators = authenticators;
```

Comment:
:squint:

Ok so - we _shouldn't_ be utilizing non lifecycle state variables like this i don't think 🤔